colaraz/lms/templates/main.html

```diff
@@ -446,9 +446,14 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
 
     <%include file="widgets/segment-io-footer.html" />
     <script type="text/javascript">
+        // Colaraz Notifications API variables
         var colarazNotificationsFetchingUrl = "${reverse('colaraz_features:notifications_handler', kwargs={'api_method': 'fetch'})}";
         var colarazNotificationsMarkingUrl = "${reverse('colaraz_features:notifications_handler', kwargs={'api_method': 'mark'})}";
-        var notificationsRefreshTime = ${ getattr(settings, 'COLARAZ_NOTIFICATIONS', {}).get('REFRESH_TIME_MILLISECONDS', 30000)};
+        var colarazNotificationsRefreshTime = ${ getattr(settings, 'COLARAZ_NOTIFICATIONS', {}).get('REFRESH_TIME_MILLISECONDS', 30000)};
```

Comment:
Make sure to use a single variable. in the template above `REFRESH_RATE_MILLISECONDS` was used and here you are using `REFRESH_TIME_MILLISECONDS`. I might be wrong if both of these variables serve different purposes.

Reply:
Updated.